schema/method_units/physical/psp.json

```diff
@@ -0,0 +1,60 @@
+{
+    "schemaId": "method-physical-psp",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Core-valence separation by means of pseudopotentials (effective potential)",
+    "allOf": [
+        {
+            "$ref": "../method_unit_base.json"
+        }
+    ],
+    "properties": {
+        "methodCategoryPaths": {
+            "type": "array",
+            "items": {
+                "anyOf": [
+                    {
+                        "enum": [
+                            "linalg/lintra"
+                        ]
+                    }
+                ]
+            }
+        },
+        "type": {
+            "enum": [
+                {
+                    "name": "Pseudopotential Approximation",
+                    "slug": "psp"
```

Comment:
I remember there was a type definition for oneOf "string" or "object" with name and slug, so that `type: "psp" would also validate, wasn't there?

Reply:
yes, there's `slugified_entry_or_slug` in `core/primitive`.